### PR TITLE
Some fixes and updates to the DOC index page

### PR DESCRIPTION
**Pull request by tdipisa**
## Description

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [x] Other... Please describe: index doc fixes and updates

## Issue

**What is the current behavior?**

#<issue>

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

## Other useful information

[//]: # (rtdbot-start)

URL of RTD document: https://mapstore.readthedocs.io/en/tdipisa-patch-4/ Documentation Status

[//]: # (rtdbot-end)

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
+[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) is not only a product but also a webgis framework. As a standard geoportal product, it is a web-based product that allows to provide
```
Comment (offtherailz) [inline docs/index.md]:
```suggestion
[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) is not only a product but also a WebGIS framework. As a standard geoportal product, it is a web-based product that allows to provide  
```

Comment (tdipisa) [reply docs/index.md]: done

File: docs/index.md [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
+[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) is not only a product but also a WebGIS framework. As a standard geoportal product, it is a web-based product that allows to provide  
+a powerfull and interactive geospatial WebGIS, it provides a direct and real-time access to geospatial data warehouses and it supports all available
```
Comment (offtherailz) [inline docs/index.md]:
```suggestion
a powerful and interactive geospatial WebGIS, it provides a direct and real-time access to geospatial data warehouses and it supports all available 
```

Comment (tdipisa) [reply docs/index.md]: @offtherailz done

**Pull request merged**